src/ring_theory/polynomial/cyclotomic.lean

```diff
@@ -643,7 +643,7 @@ section minpoly
 open is_primitive_root complex
 
 /-- The minimal polynomial of a primitive `n`-th root of unity `μ` divides `cyclotomic n ℤ`. -/
-lemma minpoly_primitive_root_dvd_cyclotomic {n : ℕ} {K : Type*} [field K] {μ : K}
+lemma _root_.minpoly_primitive_root_dvd_cyclotomic {n : ℕ} {K : Type*} [field K] {μ : K}
```

Comment:
I guess technically, this would be either `minpoly_dvd_cyclotomic` or `minpoly_dvd_cyclotomic_of_is_primitive_root`. I would prefer the shorter one.